SR2020/2020-05-26-virtual-competition-dropped-out-teams.md

```diff
@@ -0,0 +1,40 @@
+---
+to: sr2020-teams
+subject: Would you like to compete in the Student Robotics 2020 Virtual Competition
+---
+
+![](https://studentrobotics.org/docs/resources/competition-simulator/arena.png)
+
+Hello!
+
+We understand that you dropped out of Student Robotics 2020. However, given the situation regarding COVID-19 and our switch to a virtual competition, your circumstances may have changed.
```

Comment:
This sentence reads a little weird to me. 

```suggestion
We understand that were unable to compete in the Student Robotics 2020 competition. However, given the situation regarding COVID-19 and our [transition to a virtual competition}(https://studentrobotics.org/news/2020-05-22-virtual-competition-transition/), we're wondering if you may be interested in competing again?
```

Reply:
Thanks, that reads much better!